src/apps/vpn/appfeaturelistcallback.h

```diff
@@ -144,7 +144,7 @@ bool FeatureCallback_splitTunnel() {
 
 bool FeatureCallback_startOnBoot() {
 #if defined(MZ_LINUX) || defined(MZ_MACOS) || defined(MZ_WINDOWS) || \
-    defined(MZ_DUMMY) || defined(MZ_WASM) || defined(MZ_ANDROID)
+    defined(MZ_DUMMY) || defined(MZ_WASM) || defined(MZ_IOS)
```

Comment:
Is this  removing the feature from Android?

Reply:
Oooops. that should get removed, yes but not in this PR :)